os_net_config/common.py

```diff
@@ -324,6 +326,61 @@ def is_vf(pci_address):
     return is_sriov_vf
 
 
+def get_pci_address(name):
+    """Fetch the PCI address of the interface
+
+    Fetch the PCI address of the device from the /sys/class/net
+    subsystem when the interface is bound with the ethernet drivers.
+    If the device is bound with vfio-pci, the pci address is fetched
+    from the dpdk map.
+    :param name: name of the interface. For vfs the name could
+        be written in the format f"sriov:{pf_name}:{vfid}"
+    :returns: Return the PCI address
+    """
+    vfid = None
+    device = name.split(":")
+    ifname = device[0]
+    if len(device) == 3 and ifname == "sriov":
+        ifname = device[1]
+        vfid = device[2]
+
+    if vfid:
+        dev_path = get_dev_path(ifname, f'virtfn{vfid}')
+    else:
+        dev_path = get_dev_path(ifname, '_device')
```

Comment:
This is a bit obscure. Maybe we could have an explicit way to determine which path we get? Instead of relying on "_" at the beginning of the argument.